theurgy.lic

```diff
@@ -55,75 +135,296 @@ def buy_wine
+    pause 1
+    waitrt?
+    DRC.fix_standing
+  end
+end
 
-DRCM.ensure_copper_on_hand 500
+def incense_wine
+  DRC.bput("get #{$settings.flint_lighter}", 'You get', 'I could not find', 'What were you referring to')
+  get_from_container('incense')
+  while 'nothing happens' == DRC.bput('light my incense with flint', 'nothing happens', 'bursts into flames', 'much too dark in here to do that')
+    waitrt?
+  end
+  waitrt?
+  fput 'wave incense at altar'
+  fput 'snuff incense'
+  put_in_container('incense')
+  fput "stow #{$settings.flint_lighter}", 'You put your', 'Stow what'
+  pause 1
+  get_from_container('wine')
+  fput 'pour wine on altar'
+  put_in_container('wine')
+end
 
-buy_incense if 'and see' != DRC.bput("rummage /C incense my #{$settings.theurgy_supply_container}", 'but there is nothing in there like that', 'and see')
-buy_wine if 'and see' != DRC.bput("rummage /C wine my #{$settings.theurgy_supply_container}", 'but there is nothing in there like that', 'and see')
-buy_flint if 'and see' != DRC.bput("rummage /C flint my #{$settings.theurgy_supply_container}", 'but there is nothing in there like that', 'and see')
+def recite_prayer
+  fput "recite Meraud, power the holy fires that unleash my righteous vengeance;Chadatru, guide my sword to swing in justice;Everild, give me the power to conquer my enemies;Truffenyi, let me not lose sight of compassion and mercy;Else, I will become like those I despise;Urrem'tier, receive into your fetid grasp these wicked souls;May the Tamsine's realms never know their evil ways again;May all the Immortals guide your faithful soldier #{checkname}."
+  pause 1
+  waitrt?
+end
 
-DRCT.walk_to('5989')
+def reflectory
+  DRCT.walk_to '5988'
+  fput 'meditate'
+  waitrt?
+end
 
-clean_altar if 'dusty' == DRC.bput('look altar', 'dusty', 'The altar is carved of')
-waitrt?
-DRC.bput('pray meraud', 'meraud')
-waitrt?
+def invoke_parchment
+  DRCT.walk_to('1230')
+  if 'but there is nothing in there like that' != DRC.bput("rummage /C parchment my #{$settings.theurgy_supply_container}", 'but there is nothing in there like that', 'pale golden prayer parchment')
+    get_from_container('golden parchment')
+    fput 'invoke my parchment'
+    waitfor 'that you might cast your sacred light'
+    put_in_container('golden parchment')
+  end
+end
 
-fput 'remove badge'
-fput 'pray badge'
-pause 2
-waitrt?
-fput 'wear badge', 'You put on a', 'You are already'
+######################## CUSTOM TRAVEL ########################
 
-until ['flawless performance to those on high', 'In your condition'].include? DRC.bput('dance', 'flawless performance to those on high', 'You begin to dance', 'Your actions grow', 'Your dance', 'but you falt', 'In your condition')
-  pause 1
+def enter_altar
+  DRCT.walk_to('1164')
+  move 'go shack'
+  fput 'look under blanket'
+  move 'go hole'
+  move 'west'
+  move 'up'
   waitrt?
-  DRC.fix_standing
```

Comment:
probably not.  I think this assuming I'm at the right spot in my head is after the bathe - I had expected to be laying down (added the entire section while hunting) but you don't actually get put prone from this.